fix: azuread_application resource import example (#1231)
hashicorp/terraform-provider-azuread: docs/resources/application.md

> Applications can be imported using their object ID, e.g.

Reviewer: ```suggestion
Applications can be imported using their object ID, in the following format.
```
